frameworks/cassandra/src/test/java/com/mesosphere/sdk/cassandra/scheduler/ServiceSpecTest.java

```diff
@@ -33,7 +33,10 @@ public ServiceSpecTest() {
                 "TASKCFG_ALL_CASSANDRA_HEAP_NEW_MB", "400",
                 "CASSANDRA_HEAP_GC", "CMS",
                 "CASSANDRA_DISK_MB", "5000",
-                "CASSANDRA_DISK_TYPE", "ROOT");
+                "CASSANDRA_DISK_TYPE", "ROOT",
+
```

Comment:
Is the additional line required?

Reply:
I was hoping to visually separate TLS options, I can remove the extra line if you'd prefer that.

Reply:
No, I'm OK with it.